### PR TITLE
Create Tictactoe.java

### PR DESCRIPTION
**Pull request by dimloko**
Tic tac toe

### CODE REVIEW

File: tictactoe/src/Tictactoe.java [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+            out(array); //Вывод игрового поля
+            userInput(sc, array); //Ход игрока
+            if (!stepAI(array, 0, 2)) {     //Сначала ищем 0 в количестве 2 шт.
+                if (!stepAI(array, 1, 2)) { //Если не находим, то ищем 1 в количестве 2 шт
+                    stepAI(array, 1, 1);    //Если не находим, то ищем 1 в количестве 1 шт
+                }
+            }
+            if (check(array)) { //Проверка поля на выйгрыш либо наличие своьодных клеток
+                out(array);
+                System.out.println("Игра окончена");
```
Comment (TheFallen058) [inline tictactoe/src/Tictactoe.java:19]: Not clear who is a winner)

File: tictactoe/src/Tictactoe.java [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    }
+
+    //Ход игрока
+    static void userInput(Scanner sc, int[][] array) {
+
+        if (check(array)) { //Проверка количества оставшихся незаполненных клеток
+            return;
+        }
+
+        int i, j; //Строка и столбец для хода игрока
```
Comment (TheFallen058) [inline tictactoe/src/Tictactoe.java]: if you know that it is row and column you can give this variables correct name and no comment needed.

**Review by TheFallen058 — APPROVED** [review]
Amazing implementation!

File: tictactoe/src/Tictactoe.java (hunk 2 above)
Comment (dimloko) [reply tictactoe/src/Tictactoe.java]: > if you know that it is row and column you can give this variables correct name and no comment needed.

Corrected according to remark

**Pull request merged**